README.md

```diff
@@ -62,6 +62,7 @@ Explore our detailed CLV examples using data from the [`lifetimes`](https://gith
 
 - [CLV Quickstart](https://pymc-marketing.readthedocs.io/en/stable/notebooks/clv/clv_quickstart.html)
 - [BG/NBD model](https://pymc-marketing.readthedocs.io/en/stable/notebooks/clv/bg_nbd.html)
+- [Pareto model](https://pymc-marketing.readthedocs.io/en/stable/notebooks/clv/pareto_nbd.html)
```

Comment:
How about Pareto/NBD

Reply:
Good point! added in https://github.com/pymc-labs/pymc-marketing/pull/649/commits/3b29f2e79a22787e24d0d3e3b03925d82172381d